### ISSUE TITLE
loss does not converge

### ISSUE THREAD

**Issue by redorangeyellowy** [OlgaChernytska/2D-Hand-Pose-Estimation-RGB#3]
Hello. Thank you for the nice repository. There was something strange about training code through the jupyter notebook. As shown in the picture below, there was a problem that the train loss did not converge until 1000 epochs, and I cloned the code that was posted on github, but I am not sure why this problem occurs. Please give us some advice!

![스크린샷 2022-05-05 오후 1 16 16](https://user-images.githubusercontent.com/86509883/166862946-62c498cc-2dfc-4f6c-a3b9-26d57a64d276.png)

![스크린샷 2022-05-05 오후 1 16 48](https://user-images.githubusercontent.com/86509883/166862989-843a99b7-811d-4279-a5b6-bc1cdb0e7dda.png)

**Comment by dolomitt** [comment]
Not sure what you are doing wrong since it is working for me out of the box. Just showing the first 30 epochs but it continues to go down.

trainer = Trainer(model, criterion, optimizer, config, scheduler)
model = trainer.train(train_dataloader, val_dataloader)

Epoch: 1/1000, Train Loss=0.9928900471, Val Loss=0.9928812754
Epoch: 2/1000, Train Loss=0.9928321873, Val Loss=0.9928021147
Epoch: 3/1000, Train Loss=0.99276536, Val Loss=0.9927285853
Epoch: 4/1000, Train Loss=0.9926900244, Val Loss=0.9926433904
Epoch: 5/1000, Train Loss=0.9926007285, Val Loss=0.9925357614
Epoch: 6/1000, Train Loss=0.992494887, Val Loss=0.9924166884
Epoch: 7/1000, Train Loss=0.992359498, Val Loss=0.9922742219
Epoch: 8/1000, Train Loss=0.9922054924, Val Loss=0.9920888061
Epoch: 9/1000, Train Loss=0.9920186003, Val Loss=0.9918846829
Epoch: 10/1000, Train Loss=0.9917870094, Val Loss=0.9916118241
Epoch: 11/1000, Train Loss=0.9915017731, Val Loss=0.9912880432
Epoch: 12/1000, Train Loss=0.9911332165, Val Loss=0.9908700245
Epoch: 13/1000, Train Loss=0.9906246662, Val Loss=0.9902654347
Epoch: 14/1000, Train Loss=0.9898954349, Val Loss=0.9892480686
Epoch: 15/1000, Train Loss=0.9887793403, Val Loss=0.988282936
Epoch: 16/1000, Train Loss=0.9866293645, Val Loss=0.9847899647
Epoch: 17/1000, Train Loss=0.9811192844, Val Loss=0.976242951
Epoch: 18/1000, Train Loss=0.9635404126, Val Loss=0.9428994769
Epoch: 19/1000, Train Loss=0.9172500454, Val Loss=0.8996225737
Epoch: 20/1000, Train Loss=0.8709353302, Val Loss=0.8515608084
Epoch: 21/1000, Train Loss=0.846778491, Val Loss=0.8371145782
Epoch: 22/1000, Train Loss=0.8284842384, Val Loss=0.8216351044
Epoch: 23/1000, Train Loss=0.8109641741, Val Loss=0.8085817666
Epoch: 24/1000, Train Loss=0.7938039583, Val Loss=0.7909920783
Epoch: 25/1000, Train Loss=0.7782229454, Val Loss=0.8003070582
Epoch: 26/1000, Train Loss=0.766760181, Val Loss=0.76367521
Epoch: 27/1000, Train Loss=0.7518730304, Val Loss=0.7431984061
Epoch: 28/1000, Train Loss=0.7436600643, Val Loss=0.7542876715
Epoch: 29/1000, Train Loss=0.7305706459, Val Loss=0.7235489828
Epoch: 30/1000, Train Loss=0.7268596911, Val Loss=0.7286577395
